Create documentation for snapshots with hashed prefix path type (#8196)
opensearch-project/documentation-website: _install-and-configure/configuring-opensearch/index-settings.md

- `cluster.remote_store.segment.transfer_timeout` (Time unit): Controls the maximum amount of time to wait for all new segments to update after refresh to the remote store. If the upload does not complete within a specified amount of time, it throws a `SegmentUploadFailedException` error. Default is `30m`. It has a minimum constraint of `10m`.

> - `cluster.remote_store.translog.path.prefix` (String): Controls the fixed path prefix for translog data on a remote store enabled cluster. This setting is effective when `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. This defaults to `""` (empty string).

Author: It comes into effect only when `cluster.remote_store.index.path.type` setting has one of `hashed_prefix` or `hashed_infix` values.

> - `cluster.remote_store.translog.path.prefix` (String): Controls the fixed path prefix for translog data on a remote store enabled cluster. This setting is effective when `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. This defaults to `""` (empty string).

Reviewer: ```suggestion
- `cluster.remote_store.translog.path.prefix` (String): Controls the fixed path prefix for translog data on a remote store enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```

> - `cluster.remote_store.translog.path.prefix` (String): Controls the fixed path prefix for translog data on a remote store enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.

Reviewer 2: ```suggestion
- `cluster.remote_store.translog.path.prefix` (String): Controls the fixed path prefix for translog data on a remote-store-enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```

> - `cluster.remote_store.segments.path.prefix` (String): Controls the fixed path prefix for segments data on a remote store enabled cluster. This setting is effective when `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. This defaults to `""` (empty string).

Reviewer: ```suggestion
- `cluster.remote_store.segments.path.prefix` (String): Controls the fixed path prefix for segments data on a remote store enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```

> - `cluster.remote_store.segments.path.prefix` (String): Controls the fixed path prefix for segments data on a remote store enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.

Reviewer 2: ```suggestion
- `cluster.remote_store.segments.path.prefix` (String): Controls the fixed path prefix for segment data on a remote-store-enabled cluster. This setting only applies when the `cluster.remote_store.index.path.type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```

> - `cluster.snapshot.shard.path.prefix` (String): Controls the fixed path prefix for snapshot shard level blobs. This setting is effective when repository `shard_path_type` setting is either `hashed_prefix` or `hashed_infix`. This defaults to `""` (empty string).

Reviewer: ```suggestion
- `cluster.snapshot.shard.path.prefix` (String): Controls the fixed path prefix for snapshot shard level blobs. This setting only applies when the repository `shard_path_type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```

> - `cluster.snapshot.shard.path.prefix` (String): Controls the fixed path prefix for snapshot shard level blobs. This setting only applies when the repository `shard_path_type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.

Reviewer 2: ```suggestion
- `cluster.snapshot.shard.path.prefix` (String): Controls the fixed path prefix for snapshot shard-level blobs. This setting only applies when the repository `shard_path_type` setting is either `hashed_prefix` or `hashed_infix`. Default is an empty string, `""`.
```
